rest_framework/renderers.py

```diff
@@ -635,6 +635,7 @@ def get_context(self, data, accepted_media_type, renderer_context):
             'view': view,
             'request': request,
             'response': response,
+            'user': request.user,
```

Comment:
For on reason or another, yes.
I've not dug into why yet, but presumably we were previously having it automatically included by the `RequestContext`. I assume that we could also change our settings in order to resolve this (eg perhaps context processor configuration has moved to being part of the `TEMPLATES` dictionary?)